app/routes/organization/settings.js

```diff
@@ -16,7 +16,12 @@ export default TravisRoute.extend({
     if (organization.permissions.admin !== true) {
       this.transitionTo('organization.repositories', organization);
     }
-    const preferences = this.store.query('preference', { organization_id: organization.id });
-    return hash({ organization, preferences });
+    // const preferences = organization.get('preferences');
+    // this.store.query('preference', { organization_id: organization.id });
+
+    return hash({
+      organization,
+      preferences: organization.get('preferences')
```

Comment:
Hmm, It's a good question. I've looked into it a bit and I'm not sure yet what the cause is. I thought it might be something about the way the organization model is fetched, but that doesn't seem to be the case.

Reply:
Hey @emmanuel-ci, I'm not sure as well, but I'd recommend you to debug the [adapter](https://github.com/travis-ci/travis-web/blob/master/app/adapters/preference.js) for `preference` request. It seems like the adapter may treat `organization_id` as mandatory, whereas `organization.get('preferences')` doesn't provide it out of the box, it provides the list of `id`s.